types/types.go

```diff
@@ -151,10 +151,17 @@ type SpecReport struct {
 	//It includes detailed information about the Failure
 	Failure Failure
 
-	// NumAttempts captures the number of times this Spec was run.  Flakey specs can be retried with
-	// ginkgo --flake-attempts=N
+	// NumAttempts captures the number of times this Spec was run.
+	// Flakey specs can be retried with ginkgo --flake-attempts=N or the use of the FlakeAttempts decorator.
+	// Repeated specs can be retried with the use of the RepeatAttempts decorator
 	NumAttempts int
 
+	// IsFlakeAttempts captures whether the spec has been retried with ginkgo --flake-attempts=N or the use of the FlakeAttempts decorator.
```

Comment:
As mentioned in the issue I think:

```go
MaxFlakeAttempts int
RequiredPasses int
```

might be more informative and easier to work with.